cmd/agent/api/response/types.go

```diff
@@ -24,5 +24,5 @@ type TaggerListResponse struct {
 
 // TaggerListEntity holds the tagging info about an entity
 type TaggerListEntity struct {
-	Tags map[string][]string `json:"tags"`
```

Comment:
I see this has a `json` annotation -- is this changing the format of any external JSON?

Reply:
Yes, this is part of the flare payload. I'm not aware of any automated consumers (and this changed not too far back anyway, to no noise whatsoever), so I'm assuming it's only read by humans, and most likely humans working at Datadog.